inspire/base/static/js/jquery.menu-aim.js

```diff
@@ -0,0 +1,323 @@
+/**
+ * menu-aim is a jQuery plugin for dropdown menus that can differentiate
+ * between a user trying hover over a dropdown item vs trying to navigate into
```

Comment:
Instead of copy/pasting the whole file, we can add it as a dependency.

```bash
$ bower search aim
```

```bash
jQuery-menu-aim git://github.com/kamens/jQuery-menu-aim.git
```